src/lib/model/index.ts

```diff
@@ -1 +1,7 @@
+export * from './account-vo';
+export * from './archive-vo';
+export * from './folder-vo';
+export * from './record-vo';
 export * from './request-vo';
+export * from './simple-vo';
+export * from './share-by-url-vo';
```

Comment:
Alphabetize:

```suggestion
export * from './share-by-url-vo';
export * from './simple-vo';
```

(We should really set up stricter linter rules; it's such a silly thing to comment on in a code review!)